testing2x

```diff
@@ -49,7 +49,7 @@ done
 for pkgarch in ${ARCHES[@]}; do
 	repo_unlock ${TESTING_REPO} ${pkgarch}
 done
-for repo in ${STABLE_REPOS[@]}; do
+for repo in  "${STABLE_REPOS[@]}"; do
```

Comment:
Added extra whitespace here, commit message says you are removing whitespace.

Reply:
Huh.  I wonder how I screwed that up.  But I did remove whitespace further up in the commit!